src/AppBundle/Search/SearchResultsProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace AppBundle\Search;
+
+use AppBundle\Entity\Committee;
+use AppBundle\Repository\CommitteeMembershipRepository;
+use AppBundle\Repository\CommitteeRepository;
+
+class SearchResultsProvider
+{
+    private $committee;
+    private $membership;
+
+    public function __construct(CommitteeRepository $committee, CommitteeMembershipRepository $membership)
+    {
+        $this->committee = $committee;
+        $this->membership = $membership;
+    }
+
+    public function findCommittees()
+    {
+        // the following part is a fixture only, for this issue
+
+        /** @var Committee[] $committees */
+        $committees = $this->committee->findBy([], ['name' => 'asc'], 20);
+
+        foreach ($committees as $committee) {
+            $results[] = [
+                'committee' => $committee,
+                'memberships' => $this->membership->countMembers($committee->getUuid()),
+            ];
+        }
+
+        return $results ?? [];
```

Comment:
I know you are not used to this technique but this is the same, with on line of code less. Our big friend Julien Pauli have validated this way to do since the PHP 7 static array management.